lms/djangoapps/bulk_email/tests/test_models.py

```diff
@@ -179,7 +201,7 @@ def tearDown(self):
 
     def test_creation_auth_on(self):
         BulkEmailFlag.objects.create(enabled=True, require_course_email_auth=True)
-        course_id = SlashSeparatedCourseKey('abc', '123', 'doremi')
+        course_id = CourseKey.from_string('abc/123/doremi')
```

Comment:
@adampalay @dianakhuang Tests pass locally with this substitution, so it seems to work